File: apps/staff/lib/features/services/domain/entities/service.dart
class Service {
  final String id;
  final String name;
  final String? description;
  final double price;
  final int durationMinutes;
  final bool isActive;
  final int sortOrder;

  const Service({
    required this.id,
    required this.name,
    this.description,
    required this.price,
    required this.durationMinutes,
    required this.isActive,
    required this.sortOrder,
  });
}

File: apps/admin-v2/src/domain/entities/service.ts
export interface Service {
  id: string;
  name: string;
  description: string | null;
  price: number;
  isActive: boolean;
  /** Si el trabajo lleva secado. Sólo lo usan las lavadoras: decide si
      completar el registro exige asignar un secador. */
  requiresDryer: boolean;
  imageUrl: string | null;
  sortOrder: number;
  createdAt: Date;
}
